setup-scripts/health-checker.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import discord, requests, traceback, asyncio, os, sys, json, io
+from datetime import datetime, timedelta
+from bot_utils import setup, send_msg, siad, sc_precision
+
+"""
+health-checker reads the /health-check endpoint of the portal and dispatches
+messages to a Discord channel.
+"""
+
+# The default check interval in hours.
+DEFAULT_CHECK_INTERVAL = 1
+
+bot_token = setup()
+client = discord.Client()
+
+
+# get_hostname reads the HOSTNAME from the .env file passed as first argument
+# to the script
+async def get_hostname():
+    if len(sys.argv) > 1:
+        env_file = sys.argv[1]
+    with open(env_file, 'r') as file:
+        for line in file.read().split('\n'):
+            pair = line.split("=")
+            if pair[0] == "HOSTNAME":
+                return pair[1]
+    await send_msg(client, "HOSTNAME not found, cannot check health status", force_notify=True)
+    os.exit(0)
```

Comment:
In case `HOSTNAME` is not defined, we could return `localhost` and it should work just fine.